packages/desktop-client/src/components/budget/rollover/TransferTooltip.tsx

```diff
@@ -10,52 +9,21 @@ import { Button } from '../../common/Button';
 import { InitialFocus } from '../../common/InitialFocus';
 import { Input } from '../../common/Input';
 import { View } from '../../common/View';
-import { Tooltip } from '../../tooltips';
 import { addToBeBudgetedGroup } from '../util';
 
-type TransferTooltipProps = ComponentPropsWithoutRef<typeof Tooltip> & {
+type TransferTooltipProps = {
   initialAmount?: number;
   showToBeBudgeted?: boolean;
   onSubmit: (amount: number, categoryId: string) => void;
+  onClose: () => void;
 };
 
 export function TransferTooltip({
```

Comment:
We should consider renaming this as they're no longer tooltips. Maybe `TransferMenu`?